extension/js/common/settings.ts

```diff
@@ -20,11 +20,18 @@ declare const zxcvbn: Function; // tslint:disable-line:ban-types
 
 export class Settings {
 
-  static fetchAcctAliasesFromGmail = async (acctEmail: string) => {
+  static fetchAcctAliasesFromGmail = async (acctEmail: string, includeAcctAddress: boolean = false): Promise<Dict<SendAsAlias>> => {
     const response = await Google.gmail.fetchAcctAliases(acctEmail);
-    return response.sendAs
-      .filter(alias => alias.isDefault || alias.verificationStatus === 'accepted')
-      .map(alias => alias.sendAsEmail);
+    const validAliases = response.sendAs.filter(alias => (alias.isDefault as boolean) || alias.verificationStatus === 'accepted');
+    const result: Dict<SendAsAlias> = {};
+    for (const alias of validAliases) {
+      result[alias.sendAsEmail] = { name: alias.displayName, isPrimary: false };
+    }
+    if (includeAcctAddress) {
```

Comment:
I imagine this could always be true, unless you have usecases in mind that I have overlooked

Reply:
But this would overwrite the "footer" property from Gmail, so maybe always false is better? Just skip this completely?

Reply:
I removed it at all,
When we have a footer it will be better to not overwrite existing `sendAs` aliases and just update properties `isPrimary` and 'isDefault'

Reply:
I would overwrite everything, why not?

Reply:
I'm not sure we are talking about the same thing. When you receive new data from sendAs endpoint, is there any reason not to update our sendAs storage completely?

Reply:
There is no need to complicate it, I think. Every time we receive sendAs from the API, we completely update sendAs in storage.

There will be no way to set aliases in FlowCrypt otherwise.

In another PR, there will be no way to set footers either and sendAs will be used.

I don't see any need for any logic more complicated then that, but it's possible you see something I don't (because you are the one implementing it)

Reply:
> I would overwrite everything, why not?

If I remember correctly we are going to have a 'footer' property in `sendAs` object so if we overwrite everything we will lose our footer.

Reply:
We will use footers from Gmail also, in another PR.

Until then, we are using footers from 'footer' storage. But later we'll ignore the footer storage and just use sendAs, and there will be no way to change footers in FlowCrypt.

Reply:
So, if I understood everything correctly you want me to update the `sendAs` property in our storage right away in `fetchAcctAliasesFromGmail`? (I will rename it if yes)

Reply:
Sorry, I just got what you meant,
Done.

Reply:
Sure, you could do that, if you want to have a consistent place to grab new results and update storage.

Or it could do saving only, eg:
```
async updateStorageSendAs(apiResult): Promise<void> {
   // 
}
```
Which may be slightly better decoupled. Either way is good.

Reply:
we already have this method `refreshAcctAliases`, it was returning fetched aliases from Gmail before and now it returns `boolean` that indicates whether the 'sendAs' was Changed or not.

Reply:
That sounds ok, we could just use that, then pull from local storage.